sumy/utils.py

```diff
@@ -49,15 +50,20 @@ def expand_resource_path(path):
 
 
 def get_stop_words(language):
-    path = expand_resource_path("stopwords/%s.txt" % language)
-    if not exists(path):
+    try:
+        stopwords_data = pkgutil.get_data("sumy", "data/stopwords/%s.txt" % language)
```

Comment:
Or may you have suggested write code the next way?
`stopwords_data = pkgutil.get_data("sumy", expand_resource_path("stopwords/%s.txt" % language))`